src/pybind/mgr/crash/module.py

```diff
@@ -1,20 +1,34 @@
 import hashlib
-from mgr_module import MgrModule
+from mgr_module import CLICommand, CLIReadCommand, CLIWriteCommand, HandlerFuncType, MgrModule
 import datetime
 import errno
+import functools
+import inspect
 import json
 from collections import defaultdict
 from prettytable import PrettyTable
 import re
 from threading import Event, Lock
-
+from typing import Any, Dict, Optional, Tuple
 
 DATEFMT = '%Y-%m-%dT%H:%M:%S.%f'
 OLD_DATEFMT = '%Y-%m-%d %H:%M:%S.%f'
 
 MAX_WAIT = 600
 MIN_WAIT = 60
 
+
+def with_crashes(func: HandlerFuncType) -> HandlerFuncType:
```

Comment:
I would use 

```python
FuncT = TypeVar('FuncT', bound=Callable)
```

here to keep the parameters annotated.

Reply:
incorporated.